internal/perms/acl.go

```diff
@@ -123,6 +123,21 @@ func (a ACL) Allowed(r Resource, aType action.Type) (results ACLResults) {
 	return
 }
 
+// AllowedActions returns the set of actions out of a given possible set that
+// are allowed given the scope/id/type/pin. Returns nil if no match.
+func (a ACL) AllowedActions(r Resource, aTypes action.Actions) action.Actions {
```

Comment:
Curious about the naming choice of `aTypes`, why not `available` or `actions`?

Reply:
No particular reason. I changed it to `avail`.